public/menu.json

```diff
@@ -1,11 +1,32 @@
 {
-    "products": [
-      {
-          "id": "522-2222",
-          "modifiers": [{
-              "id": "3222",
-              "price": 220
-          }], 
-          "basePrice": 2222}
-    ]
-}
\ No newline at end of file
+  "products": [
+    {
+      "slug": "blackened-cajun-chicken-sandwich",
+      "basePrice": 11.99,
```

Comment:
When dealing with currency, it's generally best to deal with whole numbers in the smallest denomination (ie: in the US to use cents). This avoids weird rounding errors that can come with floats.